{
  final Throwable failure=new RuntimeException();
  context.checking(new Expectations(){
{
      one(project).getScript();
      will(returnValue(buildScript));
      one(standardOutputRedirector).on(LogLevel.QUIET);
      one(buildScript).run();
      will(throwException(failure));
      one(standardOutputRedirector).flush();
    }
  }
);
  try {
    evaluator.evaluate(project);
    fail();
  }
 catch (  RuntimeException e) {
    assertThat(e,sameInstance(failure));
  }
}
